pasha_modules/util.coffee

```diff
@@ -255,7 +233,6 @@ module.exports = {
     sendConfirmEmail: sendConfirmEmail
     sendStatusEmail: sendStatusEmail
     pagerdutyAlert: pagerdutyAlert
-    startNag: startNag
```

Comment:
Btw. we might want to refactor this file in the future to use `@` names for public functions